bootstrap.d/sys-auth.yml

```diff
@@ -0,0 +1,32 @@
+packages:
+  - name: seatd
+    labels: [aarch64]
+    architecture: '@OPTION:arch@'
+    source:
```

Comment:
Missing metadata information. While not all ports have this, we should add it when possible and I'd like new ports to do it by default. This is also the moment where you can decide if you like to be the maintainer of this port.